test/e2e/storage/drivers/in_tree.go

```diff
@@ -1035,6 +1058,183 @@ func (e *emptydirDriver) PrepareTest(f *framework.Framework) (*storageframework.
 	}, func() {}
 }
 
+// Cinder
```

Comment:
This looks like a rebase error. This PR should not add tests (back).

Reply:
Removed